src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
```

Comment:
Wrap if you have > 2 params or length > 80
```suggestion
  const { name, email } = user;
```